packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -42,7 +42,7 @@ import type { CartesianAnimationData, CartesianSeriesNodeDatum } from './cartesi
 import { CartesianSeries, CartesianSeriesMarker, CartesianSeriesNodeClickEvent } from './cartesianSeries';
 import { getMarkerConfig, updateMarker } from './markerUtil';
 
-interface MarkerSelectionDatum extends Required<CartesianSeriesNodeDatum> {
+interface MarkerSelectionDatum extends Required<Omit<CartesianSeriesNodeDatum, 'capDefaults'>> {
```

Comment:
Just wondering if it would be possible to go the other way around, and have an extension of `CartesianSeriesNodeDatum` which is the base for the affected cases, rather than modifying the unaffected cases?

Reply:
That sounds like a very good idea. We already do something similar with tooltip parameters (e.g. [AgLineSeriesTooltipRendererParams](https://github.com/ag-grid/ag-charts/blob/cbae49fb6a8b83624c10ba67112c86f392ec926e/packages/ag-charts-community/src/options/series/cartesian/lineOptions.ts#L12) extends [AgCartesianSeriesTooltipRendererParams](https://github.com/ag-grid/ag-charts/blob/cbae49fb6a8b83624c10ba67112c86f392ec926e/packages/ag-charts-community/src/options/series/cartesian/cartesianSeriesTooltipOptions.ts#L3) and [AgErrorBoundSeriesTooltipRendererParams](https://github.com/ag-grid/ag-charts/blob/cbae49fb6a8b83624c10ba67112c86f392ec926e/packages/ag-charts-community/src/options/series/cartesian/cartesianSeriesTooltipOptions.ts#L15))

Having a separate `ErrorBoundNodeDatum` which the line, bar and scatter series node types can extends would be a safer option